[2023-11-30T06:31:37.640+0800] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fantasy_premier_league_debug.delete_file_in_actual_folder manual__2023-11-29T22:31:21.689684+00:00 [queued]>
[2023-11-30T06:31:37.655+0800] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fantasy_premier_league_debug.delete_file_in_actual_folder manual__2023-11-29T22:31:21.689684+00:00 [queued]>
[2023-11-30T06:31:37.656+0800] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-11-30T06:31:37.683+0800] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): delete_file_in_actual_folder> on 2023-11-29 22:31:21.689684+00:00
[2023-11-30T06:31:37.686+0800] {standard_task_runner.py:57} INFO - Started process 27724 to run task
[2023-11-30T06:31:37.690+0800] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'fantasy_premier_league_debug', 'delete_file_in_actual_folder', 'manual__2023-11-29T22:31:21.689684+00:00', '--job-id', '52', '--raw', '--subdir', 'DAGS_FOLDER/azfar_dag.py', '--cfg-path', '/tmp/tmpt2f5hplv']
[2023-11-30T06:31:37.692+0800] {standard_task_runner.py:85} INFO - Job 52: Subtask delete_file_in_actual_folder
[2023-11-30T06:31:37.734+0800] {task_command.py:416} INFO - Running <TaskInstance: fantasy_premier_league_debug.delete_file_in_actual_folder manual__2023-11-29T22:31:21.689684+00:00 [running]> on host azfartitu.
[2023-11-30T06:31:37.812+0800] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='fantasy_premier_league_debug' AIRFLOW_CTX_TASK_ID='delete_file_in_actual_folder' AIRFLOW_CTX_EXECUTION_DATE='2023-11-29T22:31:21.689684+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-11-29T22:31:21.689684+00:00'
[2023-11-30T06:31:37.818+0800] {base.py:73} INFO - Using connection ID 'azure_blob_conn_id' for task execution.
[2023-11-30T06:31:38.321+0800] {base.py:73} INFO - Using connection ID 'azure_blob_conn_id' for task execution.
[2023-11-30T06:31:39.677+0800] {wasb.py:504} INFO - Deleted blobs: ('current_season_history/28112023/current_season_history_28112023.json',)
[2023-11-30T06:31:39.679+0800] {logging_mixin.py:154} INFO - File is deleted
[2023-11-30T06:31:41.001+0800] {logging_mixin.py:154} INFO - Folder 'current_season_history/archive/28112023/' deleted successfully.
[2023-11-30T06:31:41.003+0800] {logging_mixin.py:154} INFO - Folder is deleted
[2023-11-30T06:31:41.009+0800] {python.py:194} INFO - Done. Returned value was: None
[2023-11-30T06:31:41.027+0800] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=fantasy_premier_league_debug, task_id=delete_file_in_actual_folder, execution_date=20231129T223121, start_date=20231129T223137, end_date=20231129T223141
[2023-11-30T06:31:41.121+0800] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-30T06:31:41.155+0800] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
